x-pack/plugins/maps/public/classes/styles/vector/vector_style.tsx

```diff
@@ -697,12 +711,32 @@ export class VectorStyle implements IVectorStyle {
     return formatters ? formatters[fieldName] : null;
   };
 
+  getIconSvg(symbolId: string) {
+    const meta = this._getIconMeta(symbolId);
+    if (meta) {
+      return meta.svg;
+    }
+    return;
+  }
+
   _getSymbolId() {
     return this.arePointsSymbolizedAsCircles() || this._iconStyleProperty.isDynamic()
       ? undefined
       : (this._iconStyleProperty as StaticIconProperty).getOptions().value;
   }
 
+  _getIconMeta(
+    symbolId: string
+  ): { svg: string; label: string; iconSource: ICON_SOURCE } | undefined {
+    if (!symbolId) return;
+    const icon = this._customIcons.find(({ symbolId: value }) => value === symbolId);
+    if (icon) {
+      return { ...icon, iconSource: ICON_SOURCE.CUSTOM };
+    }
+    const symbol = getMakiSymbol(symbolId);
+    return symbol ? { ...symbol, iconSource: ICON_SOURCE.MAKI } : {};
```

Comment:
Change to `return symbol ? { ...symbol, iconSource: ICON_SOURCE.MAKI } : undefined;`